cloudify-web-monitoring-plugin/README.md

```diff
@@ -4,9 +4,9 @@ cloudify-web-monitoring-plugin
 
```

Comment:
Let's change the name of the folder from `cloudify-web-monitoring-plugin` to `web-monitoring-solution`

Reply:
done